<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QMainWindow" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>1280</width>
     <height>720</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>720</height>
    </size>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1280</width>
      <height>720</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1280</width>
      <height>720</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Text</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1251</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>1251</width>
        <height>641</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Map</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1271</width>
        <height>691</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/Map.png);</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>320</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sheepPlanet">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>330</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Rory's Sheep Planet</string>
      </property>
     </widget>
     <widget class="QPushButton" name="notGavinsPlanet">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Not Gavins Planet</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LorrainesPlanet">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>330</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Lorraines Planet</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Xavier">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Xavier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="frozenPlanet">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>330</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Frozen Planet</string>
      </property>
     </widget>
     <widget class="QPushButton" name="leprachauns">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>330</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Leprauchauns</string>
      </property>
     </widget>
     <widget class="QPushButton" name="landmines">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>330</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Planet Landmine</string>
      </property>
     </widget>
     <widget class="QPushButton" name="freedomBringer">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>500</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Freedom Bringers</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ghastlyPlanet">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>500</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ghastly Planet</string>
      </property>
     </widget>
     <widget class="QPushButton" name="moltenPlanet">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>330</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Molten Planet</string>
      </property>
     </widget>
     <widget class="QPushButton" name="overWorkedStudents">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>170</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Planet Overworked Students</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rascallyRabbits">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>330</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Rascally Rabbits</string>
      </property>
     </widget>
     <widget class="QPushButton" name="memeLord">
      <property name="geometry">
       <rect>
        <x>1180</x>
        <y>330</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Meme Lord</string>
      </property>
     </widget>
     <widget class="QPushButton" name="realRiders">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>490</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Real Riders</string>
      </property>
     </widget>
     <widget class="QPushButton" name="planetPokeMans">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>650</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Planet Pok√©Mans</string>
      </property>
     </widget>
     <widget class="QPushButton" name="catsMeow">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>640</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Cat's Meow System</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>GUI</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1271</width>
        <height>691</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Health Level</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="shipHealth">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Shield Level</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="shipShield">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="jumpNorthBtn">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Jump North</string>
      </property>
     </widget>
     <widget class="QPushButton" name="jumpWestBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Jump West</string>
      </property>
     </widget>
     <widget class="QPushButton" name="jumpEastBtn">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Jump East</string>
      </property>
     </widget>
     <widget class="QPushButton" name="jumpSouthBtn">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>200</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Jump South</string>
      </property>
     </widget>
     <widget class="QPushButton" name="atkBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Attack</string>
      </property>
     </widget>
     <widget class="QPushButton" name="shieldBtn">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>260</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Shield</string>
      </property>
     </widget>
     <widget class="QPushButton" name="invenBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>500</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Inventory</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>540</y>
        <width>256</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enemy Health</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>70</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Enemy Shield</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="eHealth">
      <property name="geometry">
       <rect>
        <x>1100</x>
        <y>30</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="eShield">
      <property name="geometry">
       <rect>
        <x>1100</x>
        <y>70</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>256</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>430</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Use Item</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>460</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>430</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Put Item</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
